blocks/library/quote/index.js

```diff
@@ -101,33 +101,17 @@ registerBlockType( 'core/quote', {
 			{
 				type: 'block',
 				blocks: [ 'core/paragraph' ],
-				transform: ( { value, citation, ...attrs } ) => {
-					const textElement = value[ 0 ];
-					if ( ! textElement ) {
-						return createBlock( 'core/paragraph', {
-							content: citation,
-						} );
+				transform: ( { value, citation } ) => {
+					// transforming an empty quote
+					if ( ( ! value || ! value.length ) && ! citation ) {
```

Comment:
you're checking `! value` here which means the value could be `undefined`. Should we check the same when we use `value.map` down here, or maybe just replace it with `map`?

unless `value` is always an array and this first check is unnecessary then?

Reply:
Nice catch @youknowriad, I also checked for undefined/null on the map.